src/components/Video.tsx

```diff
@@ -39,6 +39,7 @@ export const Video: FunctionComponent<Video> = ({
     autoPlay = true,
     className,
     thumbnail,
+    telemetryRecorder
```

Comment:
Component Video.tsx now has a required prop "telemetryRecorder" however it is not being passed in all places where the component is being used such as in `batch-changes.tsx`, please add the required telemetryRecorder prop to the Video component